src/public/about.html

```diff
@@ -19,10 +19,10 @@
         <nav>
           <ul class="no-bullets">
             <li>
-              <a href="/about/">About</a>
+              <a href="/about.html">About</a>
```

Comment:
These changes were from your previous PR #16, which means you created this branch from that branch using `git checkout -b rest-draft`. To avoid this, you should always run:

```bash
$ git checkout master
$ git pull upstream master
$ git push origin HEAD
$ git checkout -b my-new-branch
```

Following the instructions in my previous comment to rebase from master will resolve this.